repository/r.json

```diff
@@ -882,6 +882,17 @@
 			]
 		},
 		{
+			"name": "ReqsLang",
+			"details": "https://github.com/petarslovic/sublime-reqs-language",
+			"labels": ["requirements", "todo"],
```

Comment:
It is a syntax highlighting plugin for my own language, so not sure if I should add it as a label?